cli/src/template/android/app/src/main/java/com/example/moproapp/ZkemailComponent.kt

```diff
@@ -0,0 +1,286 @@
+package com.mopro.mopro_app
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.json.JSONObject
+import uniffi.mopro.generateNoirProof
+import uniffi.mopro.verifyNoirProof
+import java.io.File
+import com.mopro.mopro_app.getFilePathFromAssets
+import java.io.InputStream
+
+@Composable
+fun ZkEmailComponent() {
+    val context = LocalContext.current
+    var provingTime by remember { mutableStateOf("") }
+    var proofResult by remember { mutableStateOf("") }
+    var verificationTime by remember { mutableStateOf("") }
+    var verificationResult by remember { mutableStateOf("") }
+    var proofBytes by remember { mutableStateOf<ByteArray?>(null) }
+
+    // Status states
+    var isGeneratingProof by remember { mutableStateOf(false) }
+    var isVerifyingProof by remember { mutableStateOf(false) }
+    var statusMessage by remember { mutableStateOf("Ready to generate proof") }
+
+    val circuitFile = getFilePathFromAssets("zkemail.json")
```

Comment:
no such file is found
```sh
FATAL EXCEPTION: main
    Process: com.example.moproapp, PID: 27791
    java.io.FileNotFoundException: zkemail.json
                at android.content.res.AssetManager.nativeOpenAsset(Native Method)
                at android.content.res.AssetManager.open(AssetManager.java:927)
                at android.content.res.AssetManager.open(AssetManager.java:904)
                at com.example.moproapp.MainActivityKt.getFilePathFromAssets(MainActivity.kt:36)
                at com.example.moproapp.ZkemailComponentKt.ZkEmailComponent(ZkemailComponent.kt:43)
```

Reply:
fixed in [fc8bd82](https://github.com/zkmopro/mopro/pull/399/commits/fc8bd822cdf3b5c8cc5be9abac87752e89e6e4f5)